patterns/footer-1.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Title: WooCommerce Footer 1
+ * Slug: woocommerce-blocks/footer-1
+ * Categories: WooCommerce
+ * Block Types: core/template-part/footer
+ */
+?>
+
+<!-- wp:group {"tagName":"footer","layout":{"type":"default"}} -->
+<footer class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"0","padding":{"top":"0","right":"var:preset|spacing|40","bottom":"0","left":"var:preset|spacing|40"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group" style="padding-top:0;padding-right:var(--wp--preset--spacing--40);padding-bottom:0;padding-left:var(--wp--preset--spacing--40)"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group"><!-- wp:site-logo {"style":{"color":{"duotone":"var:preset|duotone|dark-grayscale"}}} /-->
+
+<!-- wp:navigation {"ref":242} /--></div>
```

Comment:
We need to make sure we don't include any ids or refs in patterns, they will not exist in other stores.
Here we can just use `<!-- wp:navigation /-->`.

Reply:
Addressed in 4b97232.